src/components/PublicHeader.spec.tsx

```diff
@@ -19,32 +17,32 @@ describe('PublicHeader', () => {
     });
 
     it('contains no login button when logged in', () => {
-        mount(<PublicHeader user={ dummyUser }/>);
+        mountWithProviders(<PublicHeader user={ dummyUser }/>);
         cy.contains('misc.publicHeader.login').should('not.exist');
     });
 
     it('contains user name if logged in', () => {
-        mount(<PublicHeader user={ dummyUser }/>);
+        mountWithProviders(<PublicHeader user={ dummyUser }/>);
         cy.contains(`${dummyUser.first_name} ${dummyUser.last_name}`);
     });
 
     it('contains user avatar if logged in', () => {
-        mount(<PublicHeader user={ dummyUser }/>);
+        mountWithProviders(<PublicHeader user={ dummyUser }/>);
```

Comment:
In this test (or a separate one) we should also verify that the logout button is rendered.